apps/communications/dialer/test/marionette/lib/views/contact_list/accessors.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
I think there's no need to create this file. The contact list is actually the contact app embedded in Dialer. I'd just use the contact page class.